code/datums/elements/spooky.dm

```diff
@@ -3,81 +3,94 @@
 	argument_hash_start_idx = 2
 	///will it spawn a new instrument
 	var/too_spooky = TRUE
-	///Once used, the element is detached
+	///If, once someone is skeletonized, the element is detached
 	var/single_use = FALSE
+	///The base multiplier of stamina damage applied by the item
+	var/stam_damage_mult
 
-/datum/element/spooky/Attach(datum/target, too_spooky = TRUE, single_use = FALSE)
+/datum/element/spooky/Attach(datum/target, too_spooky = TRUE, single_use = FALSE, stam_damage_mult = 1)
 	. = ..()
 	if(!isitem(target))
 		return ELEMENT_INCOMPATIBLE
 	src.too_spooky = too_spooky
 	src.single_use = single_use
+	src.stam_damage_mult = stam_damage_mult
 	RegisterSignal(target, COMSIG_ITEM_ATTACK, PROC_REF(spectral_attack))
 
 /datum/element/spooky/Detach(datum/source)
 	UnregisterSignal(source, COMSIG_ITEM_ATTACK)
 	return ..()
 
-/datum/element/spooky/proc/spectral_attack(datum/source, mob/living/carbon/C, mob/user)
+/datum/element/spooky/proc/spectral_attack(datum/source, mob/living/carbon/target, mob/user)
 	SIGNAL_HANDLER
 
-	if(ishuman(user)) //this weapon wasn't meant for mortals.
-		var/mob/living/carbon/human/U = user
-		if(!istype(U.dna.species, /datum/species/skeleton))
-			U.adjustStaminaLoss(35) //Extra Damage
-			U.set_jitter_if_lower(70 SECONDS)
-			U.set_stutter(40 SECONDS)
-			if(U.getStaminaLoss() > 95)
-				to_chat(U, "<font color ='red', size ='4'><B>Your ears weren't meant for this spectral sound.</B></font>")
-				INVOKE_ASYNC(src, PROC_REF(spectral_change), U)
-			if(single_use)
-				to_chat(user, span_warning("You feel like [source] has lost its spookiness..."))
-				Detach(source)
-			return
+	if(ishuman(user) && !isskeleton(target)) //this weapon wasn't meant for mortals.
+		var/mob/living/carbon/human/human = user
+		if(rattle_bones(human, stam_dam_mult = stam_damage_mult * 2))
+			to_chat(human, span_userdanger("Your ears weren't meant for this spectral sound."))
+			INVOKE_ASYNC(src, PROC_REF(spectral_change), human, user, source)
+		return
```

Comment:
I don't understand why this proc is organized as it is, it does not appear to behave anything like the old code (humans cannot be spectral blasted, it seems to me that ONLY skeletons could be?

Reply:
ah excuse me I messed up user vs target (the human var name is not all that clear apologies)